### DISCUSSION TITLE
Alist外链问题求助

### DISCUSSION THREAD

**Discussion by wang1680** [AlistGo/alist#6024] [Q&A] [answered]
我的Alist的复制链接也就是外链总是后缀带一长串代码，比如http://xxx.xxx.x.xxx:5244/d/drpy/404.jpg?sign=FJT5DzBgIlDhQ5mD8hNNfA98fNe5NGYGhqAi5BaLh7I=:0，这个是什么原因有大佬指点下么？我认识别的朋友的Alist复制链接是http://xxx.xxx.x.xxx:5244/d/drpy/404.jpg，何解呀，我反复我重新安装也没有解决，并且外链还无效，没有数据，求高人指点迷津。

**Comment by Nahuimi** [comment, marked as answer]
你是开启了签名了吧，或者是元信息那里添加了这个网盘

**Reply by wang1680** [reply]
是的，添加了元信息的，原来是这个原因，谢谢了
